Codec/Archive/Zip/Internal.hs

```diff
@@ -58,6 +59,9 @@ import qualified Data.Set            as E
 import qualified Data.Text           as T
 import qualified Data.Text.Encoding  as T
 
+blindPath :: Path Abs File    
+blindPath = $(mkAbsFile "/blindPath")
```

Comment:
Why this name? Also, it seems hackish to define it here and then even show to user. What if I had a file named `/blindPath`? How do I tell if the library is referring to that normal file or to a blind archive? Perhaps we should not use `Either` for the purpose of telling type of file we operate on, but instead we should introduce something dedicated like

```haskell
data ZipFile = ZipFileNormal (Path Abs File) | ZipFileBlind Handler
```

…and then `EntryDoesNotExist` should take `ZipFile` as argument, etc. This looks like how should be done properly, but I don't like the complexity it would introduce.